### PR TITLE
[Ready] Landing

### PR DESCRIPTION
**Pull request by woowenjun99**

### CODE REVIEW

**Comment by woowenjun99** [comment]
This is just a landing page used for the fintech website.

File: src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -1,30 +1,84 @@
...
+        }
+      },
     }),
-    // ...add more providers here
   ],
+
+  secret: env.NEXTAUTH_SECRET,
+  session: {
+    strategy: 'jwt',
+    maxAge: 7 * 60 * 60, // <--- The user can only login for 1 hour
```
Comment (xianlinc) [inline src/pages/api/auth/[...nextauth].ts]: Minor detail but did you mean 7 hours instead of 1 hour in your comment?

**Review by xianlinc — APPROVED** [review]
Overall LGTM, just some minor clarifications needed.

File: src/pages/api/auth/[...nextauth].ts (hunk 1 above)
Comment (woowenjun99) [reply src/pages/api/auth/[...nextauth].ts]: That is true haha I have changed it

**Pull request merged**